### PR TITLE
Provide visual feedback when list entry is tapped

### PR DESCRIPTION
**Pull request by hayribakici**
This PR adds a small visual feedback when clicking on a list item (app, app-list and options menu). 
The `width` of each list item had to be changed to `wrap_content` in order to let the animation look more appealing. Plus, the animation scales the text down from `1.0` to `0.85`. I am not sure if this is the "material design" approach, as I have seen that the text animation scales up (like `1.2` or so) instead of down. I made two screencasts with the down- and upscaling. Are there any preferences or knowledge about this?

Downscaling animation:
 

Upscaling animation:

### CODE REVIEW

File: app/src/main/res/xml/animate_scale.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<animated-selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <!-- the pressed state; increase x and y size to 150% -->
```
Comment (jkuester) [inline app/src/main/res/xml/animate_scale.xml:4]:
```suggestion
    <!-- the pressed state; decrease x and y size to 85% -->
```

Minor, but this comment seems inaccurate.

Comment (jkuester) [inline app/src/main/res/xml/animate_scale.xml:4]:
```suggestion
    <!-- the pressed state; decrease x and y size to 85% -->
```

Seems like this comment does not match the functionality.

**Review by jkuester — APPROVED** [review]
Looks great!

File: app/src/main/res/xml/animate_scale.xml (hunk 1 above)
Comment (hayribakici) [reply app/src/main/res/xml/animate_scale.xml]: Oops, my Bad. Thank you!

**Comment by hayribakici** [comment]
@jkuester should i do the merging?

**Comment by jkuester** [comment]
I clicked the button, so this should merge automatically as soon as the build finishes. :+1:

**Pull request merged**